Usage.md

```diff
@@ -0,0 +1,235 @@
+# How to Use the IntOGen Pipeline on Seqera
```

Comment:
this is good. I suggest adding a run example with TCGA data as input and walking through each step around how you run it and which links to use.

Reply:
Thank you for the suggestions!